src/components/Avatar.tsx

```diff
@@ -27,25 +26,31 @@ const AVATAR_COLORS = [
     '#5e5e5e',
 ]
 
-/** @typedef {'xxs' | 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl' | 'xxxl'} AvatarSize */
+type AvatarSize = 'xxs' | 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl' | 'xxxl'
 
-/** @type {AvatarSize[]} */
-const AVATAR_SIZES = ['xxs', 'xs', 's', 'm', 'l', 'xl', 'xxl', 'xxxl']
+const AVATAR_SIZES: AvatarSize[] = [
+    'xxs',
+    'xs',
+    's',
+    'm',
+    'l',
+    'xl',
+    'xxl',
+    'xxxl',
+]
 
-/**
- * @typedef {Object} Props
- * @property {string | undefined} [className]
- * @property {string[] | undefined} [colorList]
- * @property {AvatarSize | undefined} [size]
- * @property {string | undefined} [avatarUrl]
- * @property {{name?: string, email?: string}} user
- */
+interface Props {
+    className?: string
+    colorList?: string[]
+    size?: AvatarSize
+    avatarUrl?: string
+    user: { name?: string; email?: string }
+}
 
-/** @type {React.FC<Props>} */
-const Avatar = ({
+const Avatar: React.FC<Props> = ({
```

Comment:
I have done this for the 3 affected components & will make sure this is the style followed for the rest. Is this more a stylistic choice or is there any other code advantage?

Reply:
I think style we use in Twist is more readable and less typing:

```
const Avatar = ({
    user,
    avatarUrl,
    size = 'l',
    className,
    colorList = AVATAR_COLORS,
}: Props) => {
```

Reply:
I noticed there is some hoisting advantages as well, and there is a linter rule too in case we'd like to enable it: https://eslint.org/docs/rules/func-style